SwifterSwift.xcodeproj/xcshareddata/xcschemes/SwifterSwift-iOS.xcscheme

```diff
@@ -26,7 +26,8 @@
       buildConfiguration = "Debug"
       selectedDebuggerIdentifier = "Xcode.DebuggerFoundation.Debugger.LLDB"
       selectedLauncherIdentifier = "Xcode.DebuggerFoundation.Launcher.LLDB"
-      language = ""
+      language = "en"
+      region = "US"
```

Comment:
Any reason why we've modified language/region here? I don't think its a big problem but curious if we can throw it out

Reply:
I'm from the Netherlands, so for instance the `FloatExtensionsTests.testAsLocaleCurrency()` test would fail on my machine, because the result would not contain `10.23`:
```swift
XCTAssert(num.asLocaleCurrency.contains("\(num)"))
```
Actual result: `"€10,23"`

Reply:
Thanks for clarifying @basvankuijck. I think we had this problem before and I'm wondering if we just ignored it. Looks fine to me